test/unit/utils_test.rb

```diff
@@ -2,7 +2,7 @@
 
 class UtilsTest < ActiveSupport::TestCase
   include Utils
-
+  
```

Comment:
We really need to setup overcommit in our project. I have filed an issue for it. https://github.com/publiclab/plots2/issues/3967

But never mind, it's fine to commit it like this for now.